src/Polytopes/LinearProgram.jl

```diff
@@ -73,6 +82,11 @@ function objective_function(lp::LinearProgram; as::Symbol = :pair)
    end
 end
 
+"""
+    feasible_region(lp::LinearProgram)
+
+Return the feasible region of the linear program lp, which is a `Polyhedron`.
+"""
 feasible_region(lp::LinearProgram) = lp.feasible_region
```

Comment:
Just wondering: you changed `lp` to `LP` in many places, but not all. Is that intentional? I mean, it is fine by me, just wanted to draw your attention to it. It's fine by me to leave it as-is (and/or address it in a future PR), just sayin'.